src/Misc/layoutbin/en-US/strings.json

```diff
@@ -256,6 +256,7 @@
   "ServiceStartedSuccessfully": "Service {0} started successfully",
   "SessionCreateFailed": "Failed to create session.",
   "SessionExist": "A session for this agent already exists.",
+  "ShouldBeAdmin": "Access has been denied. Please make sure to run the command as an administrator.",
   "Skipping": "Does not exist. Skipping ",
```

Comment:
change to "Access denied. Please make sure to run the command as an administrator."